component/src/ImageList.test.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import ImageList from './ImageList.js';
+import renderer from 'react-test-renderer';
+
+it(' ImageList.js renders correctly', () => {
+  const tree = renderer
+    .create(< ImageList.js />)
```

Comment:
so the reason why your tests are erroring out is because of this:
change this:
.create(< ImageList.js />)
to this:
.create(<ImageList />)
you dont need the .js at the end of it basically